tests/programs/test_zeroize.gdb

```diff
@@ -33,17 +33,19 @@
 
 set confirm off
 
+# TF-PSA-Crypto uses a different name for the executable and also a build out
+# of tree.
 if $_isvoid($is_tf_psa_crypto)
     file ./programs/test/zeroize
+    set args ./framework/tests/programs/zeroize.c
 else
     file ./programs/test/tf_psa_crypto_zeroize
+    set args ../framework/tests/programs/zeroize.c
```

Comment:
If I understand correctly the ".." is because of the out-of-tree build which is not really something we should care about here. Thus I would suggest to rather build in tree in `test_tf_psa_crypto_zeroize`.